tests/Simplify/StringTest.elm

```diff
@@ -71,6 +71,118 @@ a = String.isEmpty "a"
 a = False
 """
                         ]
+        , test "should replace comparisons to the empty string with String.isEmpty (reverse order)" <|
+            \() ->
+                """module A exposing (..)
+
+a = "" == x"""
+                    |> Review.Test.run ruleWithDefaults
+                    |> Review.Test.expectErrors
+                        [ Review.Test.error
+                            { message = "Comparison with an empty string can be replaced by a call to String.isEmpty"
+                            , details = [ "You can replace this comparison to an empty string with a call to String.isEmpty, which is more efficient." ]
```

Comment:
I'm honestly not sure that it is more efficient for strings.

I do wonder if we'll get some complaints for this string simplification, I myself am not sure I like it that much :thinking: